lib/test-worker.js

```diff
@@ -34,73 +30,40 @@ sourceMapSupport.install({
 		if (sourceMapCache[source]) {
 			return {
 				url: source,
-				map: sourceMapCache[source]
+				map: fs.readFileSync(sourceMapCache[source], 'utf8')
```

Comment:
Two things:

1. It seems like this should be moved into the compiler.
2. Why are we writing maps to disk just to read them again. Wouldn't it be way faster to just not write them in the first place, and get them from memory here?